dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -2904,6 +2905,14 @@ uint32_t setAggOp(AggregateColumn* ac, Item_sum* isp)
       return rc;
     }
 
+    case Item_sum::JSON_ARRAYAGG_FUNC:
+    {
+      Item_func_group_concat* gc = (Item_func_group_concat*)isp;
```

Comment:
JSON_ARRAYAGG_FUNC uses Item_func_group_concat?

Reply:
My bad, missed it.